nautobot/circuits/templates/circuits/providernetwork.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'generic/object_detail.html' %}
+{% load static %}
+{% load helpers %}
+{% load plugins %}
+
+{% block breadcrumbs %}
+  <li><a href="{% url 'circuits:providernetwork_list' %}">Provider Networks</a></li>
+  <li><a href="{% url 'circuits:providernetwork_list' %}?provider_id={{ object.provider_id }}">{{ object.provider }}</a></li>
+  <li>{{ object }}</li>
+{% endblock %}
+
+{% block content %}
```

Comment:
It would be preferable to override the `content_left_page`, `content_right_page`, and/or `content_full_width_page` sub-blocks rather than the entire `content` block, as that reduces the amount of boilerplate required in this file.

Reply:
Implemetend!